<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="firstCamLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Параметры первой камеры</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="firstCamList">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Широта видеокамеры:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Долгота видеокамеры:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Высота видеокамеры:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Азимут:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Угол места:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Число пикселей по горизонтали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Число пикселей по вертикали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Расположение шарика по оси абцис:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Расположение шарика по оси ординат:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Размер по горизонтали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Размер по вертикали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ширина поля зрения:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Высота поля зрения:  -</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="firstCamButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Загрузить первую камеру</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="secondCamLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Параметры второй камеры</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="secondCamList">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Широта видеокамеры:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Долгота видеокамеры:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Высота видеокамеры:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Азимут:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Угол места:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Число пикселей по горизонтали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Число пикселей по вертикали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Расположение шарика по оси абцис:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Расположение шарика по оси ординат:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Размер по горизонтали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Размер по вертикали:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ширина поля зрения:  -</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Высота поля зрения:  -</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="secondCamButton">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Загрузить вторую камеру</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="resultList">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>125</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Широта:  -</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Долгота:  -</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высота:  - </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ширина:  -</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Высота:  -</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionProgramm"/>
    <addaction name="actionQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNew_Sharik">
   <property name="text">
    <string>New Ball</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionProgramm">
   <property name="text">
    <string>Program</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
